ONSdigital/eq-survey-runner
# Response guidance

## Changes
- Guidance now has show/hide toggle
- Added test for guidance
## Test
- `npm run test` 
- Confirm test passes
- Navigate to questionnaire and click on 'Show further guidance' toggle.
- Click again to close
